### ISSUE TITLE
Feature request: navigation buttons in Zotero

### ISSUE THREAD

**Issue by dbobak** [zotero/zotero#803]
As in subject - it would be nice and very convenient to have back and forward buttons allowing navigation through history of clicked items and collections. There is an extension "Zotero item history", but it is not functioning in current version of Zotero Standalone.

**Comment by aurimasv** [comment]
please post feature requests to the forums. This issue tracker is for developer use only.

**Comment by dbobak** [comment]
I have done it already: https://forums.zotero.org/discussion/50686/feature-request-navigation-buttons-in-zotero/

**Comment by dstillman** [comment]
I haven't looked at Zotero Item History, but I'd open to this feature in Zotero proper. Not sure about UI, but in Thunderbird it's nicely bound to the [ and ] keys.

**Comment by bwiernik** [comment]
My experience with Zotero Item History is that it is somewhat confusing to use. It seems to create separate history lists for each collection/library view. So clicking "Back" doesn't always go to the most previous item. I can see the potential utility, but found it more trouble than it was worth.

**Comment by dstillman** [comment]
Our implementation would certainly follow the view history.

**Comment by dstillman** [comment]
(I will say that I never actually remember to use it in Thunderbird, but it works nicely.)

**Comment by fbennett** [comment]
@bwiernik: Yeah, ZIH was an experiment. There was a bit of serendipity at the time it was written - a user with a project in progress had a need for collection-based history, and I was thinking along the same lines. Worked out well on the day, but that use case is very narrow, and the code is unfortunately not worth maintaining.
